.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,109 @@
+                'X-BLT-API-TOKEN': process.env.BLT_API_TOKEN
+              },
+              body: JSON.stringify({
+                issue_number: parseInt(process.env.ISSUE_NUMBER),
+                repo: context.repo.repo,
+                owner: context.repo.owner,
+                bounty_amount: Math.round(parseFloat(process.env.BOUNTY_AMOUNT) * 100) / 100,
+                contributor_username: process.env.CONTRIBUTOR_USERNAME,
+                pr_number: parseInt(process.env.PR_NUMBER)
+              })
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)